<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>胡志远UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>721</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>41</y>
      <width>221</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <widget class="QComboBox" name="SerialList">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>10</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>61</width>
            <height>39</height>
           </rect>
          </property>
          <property name="text">
           <string>串口号</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Connect_glove">
         <property name="text">
          <string>连接数据手套</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartGain">
         <property name="text">
          <string>开始采集数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>手套和手臂校正</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Vrep">
       <property name="text">
        <string>VREP初始化</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="AnglesList">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>310</y>
      <width>256</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="updateAngles">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>530</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>更新角度</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
